docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
```

Comment:
How about:
"After setting up and configuring your models so that they are registered or revision support you can start using the ``aldryn-reversion`` end-user features."
?